plugins/services/src/js/components/forms/PlacementSection.js

```diff
@@ -0,0 +1,270 @@
+import React, { Component } from "react";
+import { Tooltip } from "reactjs-components";
+
+import { findNestedPropertyInObject } from "#SRC/js/utils/Util";
+import AddButton from "#SRC/js/components/form/AddButton";
+import DeleteRowButton from "#SRC/js/components/form/DeleteRowButton";
+import FieldAutofocus from "#SRC/js/components/form/FieldAutofocus";
+import FieldError from "#SRC/js/components/form/FieldError";
+import FieldHelp from "#SRC/js/components/form/FieldHelp";
+import FieldInput from "#SRC/js/components/form/FieldInput";
+import FieldLabel from "#SRC/js/components/form/FieldLabel";
+import FieldSelect from "#SRC/js/components/form/FieldSelect";
+import FormGroup from "#SRC/js/components/form/FormGroup";
+import FormGroupHeading from "#SRC/js/components/form/FormGroupHeading";
+import FormGroupHeadingContent
+  from "#SRC/js/components/form/FormGroupHeadingContent";
+import FormRow from "#SRC/js/components/form/FormRow";
+import Icon from "#SRC/js/components/Icon";
+
+import OperatorTypes from "../../constants/OperatorTypes";
+import PlacementConstraintsUtil from "../../utils/PlacementConstraintsUtil";
+
+function placementConstraintLabel(name, tooltipText, options = {}) {
+  const { isRequired = false, linkText = "More information" } = options;
```

Comment:
why not move `isRequired = false, linkText = "More information"` to line `23` where `options = {}`?

Reply:
I usually try to avoid abusing function arguments section and put all destructuring assignments in the first lines. Again, please note this is an extraction. If we want to do a refactoring that should be a separate PR, didn't want to get carried away.